### PR TITLE
Previously applied orders added to state

### PR DESCRIPTION
**Pull request by Wulfheart**
Closes #125

### CODE REVIEW

File: state/state.go [hunk 1]
```diff
@@ -257,6 +260,10 @@ func (self *State) Next() (err error) {
 	return
 }
 
+func (self *State) PreviouslyAppliedOrders() map[godip.Province]godip.Adjudicator{
```
Comment (zond) [inline state/state.go:263]: Please add a comment specifying exactly what this means, when it's empty, and when it's populated.

**Comment by zond** [comment]
Looks good in principle. I don't really like how this works (since it's not populated at all until you call Next, and then it just means "orders applied at last 'Next' call"), I'd prefer a simpler solution. But I don't really know what that should look like - adding an Orders function to Phase, that's only populated post resolution, is also messy. Let's do it this way for now.

Just fix my comment, and I'll approve it!

File: state/state.go (hunk 1 above)
Comment (Wulfheart) [reply state/state.go:266]: Have a look at it if you find it sufficient. 😄

**Comment by Wulfheart** [comment]
Added the following comment:

````
// PreviouslyAppliedOrders contains all the orders which were applied during the processing of state.Next().
// If it is empty there are no previous orders yet as you probably have not run state.Next() or instantiated the state otherwise.
// Note that the orders specified here do not necessarily succeed. Again: This function only lists the orders which were applied during the processing of state.Next().
````

**Comment by zond** [comment]
👍

**Pull request merged**